### DISCUSSION TITLE
Using Frigate to post-process / re-process video files?

### DISCUSSION THREAD

**Discussion by jkalucki** [blakeblackshear/frigate#5828] [Ask A Question] [answered]
It seems like it should be possible, perhaps with some effort, to give an existing recording (i.e. a video file rather than a camera stream) to Frigate to re-process with different parameters to look for different events. Basically, to re-use Frigate's existing video processing and object detection pipeline on its own recordings. Has anyone documented such an effort?

Thanks,
John

**Comment by NickM-27** [comment, marked as answer]
This is covered in the docs https://deploy-preview-4055--frigate-docs.netlify.app/development/contributing#2-modify-your-local-config-file-for-testing

**Reply by jkalucki** [reply]
Thanks!

I looked all through the main documentation but didn't see anything related.

https://docs.frigate.video/configuration/index
https://docs.frigate.video/configuration/cameras 
and 
https://docs.frigate.video/configuration/camera_specific 
don't give any hints that the cameras: ffmpeg: inputs: path: could be file.

**Reply by NickM-27** [reply]
Frigate supports any input that ffmpeg supports
